community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexAccessor.java

```diff
@@ -149,8 +149,9 @@ public void close() throws Exception
     @Override
     public ResourceIterator<File> snapshotFiles() throws IOException
     {
-        return concatResourceIterators(
-                iterator( instanceSelector.instancesAs( new ResourceIterator[INSTANCE_COUNT], accessor -> accessor.snapshotFiles() ) ) );
+        ResourceIterator[] items = instanceSelector.instancesAs( new ResourceIterator[INSTANCE_COUNT], IndexAccessor::snapshotFiles );
```

Comment:
intentional change?

Reply:
Yes. It was messing with my IntelliJ for some reason claiming compilation issues, even though it builds fine from the command line.